src/main/java/com/epam/test_generator/services/CaseService.java

```diff
@@ -144,18 +182,37 @@ public void removeCase(Long projectId, Long suitId, Long caseId) {
         caseDAO.delete(caseId);
 
         caseVersionDAO.delete(caze);
+
+        CaseDTO removedCaseDTO = caseTransformer.toDto(caze);
+
+        return removedCaseDTO;
     }
 
-    public void removeCases(Long projectId, Long suitId, List<Long> caseIds) {
+    /**
+     * Deletes multiple cases by ids
+     * @param projectId id of project where to delete cases
+     * @param suitId id of suit where to delete cases
+     * @param caseIds list of cases ids to delete
+     * @return removed list of {@link CaseDTO}
+     */
+    public List<CaseDTO> removeCases(Long projectId, Long suitId, List<Long> caseIds) {
```

Comment:
Please test this method thoroughly, because usage of stream api along with hibernate managed entities is tricky deal.
If your testing results would not find any problems, let me know in reply to this comment.

Reply:
We tested this method with all possible data combinations via the swagger. We also debugged it and checked the results in the database. All is well and side effects haven't been found.

Reply:
Good